Firmware/UI/Screens/WhiteBalanceScreen.cpp

```diff
@@ -11,7 +11,7 @@
 #include "../../Media/Icons/home_icon.h"
 
 WhiteBalanceScreen::WhiteBalanceScreen(IUSBDevice* usbDevice) :
-    IScreen(usbDevice), _cancelButton("Cancel"), _addPresetButton("Add"), _setButton("Set"), _homeButton((Icon*)&home)
+    IScreen(usbDevice), _cancelButton("Cancel"), _addPresetButton("Add"), _setButton("Set"), _homeButton((Icon*)&home_icon)
```

Comment:
Why is the cast still there? Haven't we done the changes to avoid it?

Reply:
Yes